e2e/LongHaul/prerequisites/LongHaulSetup.ps1

```diff
@@ -177,7 +184,9 @@ az deployment group create `
     --only-show-errors `
     --template-file "$PSScriptRoot\longhaul-resources.json" `
     --parameters `
+    UserObjectId=$userObjectId `
```

Comment:
Why do we need this?

Reply:
it's been used by [keyvault](https://github.com/Azure/azure-iot-sdk-csharp/blob/brycewang/settings/e2e/LongHaul/prerequisites/longhaul-resources.json#L126).

Reply:
Oh, to give the current user permissions to the key vault secrets. Got it.